src/main/java/ladder/domain/ResultUsers.java

```diff
@@ -0,0 +1,44 @@
+package ladder.domain;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.Collectors;
+import java.util.stream.IntStream;
+
+public class ResultUsers {
+    private final Map<User, String> result;
```

Comment:
result의 value에도 String 대신 문자열을 포장한 객체를 활용할 수 있을 것 같아요
문자열을 포장해서 불변 객체로 활용하는 것도 충분히 의미가 있을 것 같은데 영훈님 생각은 어떠신가요?
```suggestion
    private final Map<User, Result> result;
```